activesupport/test/inflector_test.rb

```diff
@@ -157,6 +157,10 @@ def test_acronyms
     end
   end
 
+  def test_capitalize_first
+    assert_equal "What a Lovely Day", ActiveSupport::Inflector.capitalize_first("what a Lovely Day")
```

Comment:
please also add test for the delegation, `assert_equal "What a Lovely Day", "what a Lovely Day".capitalize_first`, as that will likely be more common usage than manually using the inflector.